cranelift/isle/docs/language-reference.md

```diff
@@ -14,9 +14,9 @@ Documentation on how ISLE is used in Cranelift can be found
 ## Intro and Whirlwind Tour: DSL for Instruction Lowering
 
 The goal of ISLE is to represent *instruction lowering patterns*. An
-instruction lowering pattern is a specification that a certain
-combination of operators in the IR (CLIF), when combined under certain
-conditions, can be compiled down into a certain sequence of machine
+instruction lowering pattern is a specification describing certain
+operators in the IR (CLIF) when combined under certain
+conditions, can be compiled down to a certain sequence of machine
```

Comment:
This rewording is incorrect. "A ... pattern is a specification that a combination of operators ... can be compiled" can't be rewritten into "A ... pattern is a specification describing certain operators ... can be compiled". The first (current) form has a subordinate clause describing what the specification does, and the verb indicates that this specification can be compiled. Your rewrite somehow attaches two actions onto "specification" -- it describes, and it can be compiled. These aren't semantically equivalent.